### ISSUE TITLE
Install

### ISSUE THREAD

**Issue by AugustMiller** [AugustMiller/rick#3]
Rick should be able to pull down the current version of a specified Kirby distribution, like [Plainkit](https://github.com/getkirby/plainkit), [Langkit](https://github.com/getkirby/langkit) or [Starterkit](https://github.com/getkirby/starterkit), and optionally include the [Panel](https://github.com/getkirby/panel), [Kirby core](https://github.com/getkirby/kirby) and [Toolkit](https://github.com/getkirby/kirby) as submodules, recursively.
- [ ] Download blob
- [ ] Unzip
- [ ] Initialize git repository
- [ ] Use submodules to track `panel` and `kirby`
- [ ] Create Directory `assets/avatars`
- [ ] Create Directory `thumbs`
- [ ] Create Directory `site/accounts`

**Comment by jackjennings** [comment]
Seems like there are two ways to go with this:
1. Clone the three various starter kits, or
2. Scaffold a basic structure and then optionally create submodule

Maybe we can just start by having the command clone plainkit, since that seems to be the most bare-bones starting option.

**Comment by AugustMiller** [comment]
Seuberling just mentioned that you can clone and replace the `remote` to basically have a duplicate. No need to reinitialize anything:

```
git clone --recursive git@github.com/getkirby/plainkit.git
git remote set-url origin https://github.com/username/new-repo.git
```

**Comment by jackjennings** [comment]
True, but the git history comes with it as well, which might not be desirable.

**Comment by AugustMiller** [comment]
I don't really have any objections to that, from a technical perspective— the same goes for when you pull down the modified WordPress install for Google App Engine or Pantheon… From a tidiness point of view, though, I do like to have a repo be all my own/collaborators' commits. :smiling_imp:

**Comment by AugustMiller** [comment]
http://stackoverflow.com/a/23486788/2777954

> This is useful to create the "initial commit" in a new project using another repository as the template/archetype/seed/skeleton.

There are actually a number of nice solutions in that thread.

**Comment by jackjennings** [comment]
The actual command for this would be `rick new [path]`, right?

**Comment by AugustMiller** [comment]
We're not really creating a new Rick project, it's a new Kirby project… Maybe `rick create [path]`?

**Comment by AugustMiller** [comment]
One task that should be part of this process is the creation of the `assets/avatars`, `site/accounts` and `thumbs` directories.

**Comment by jackjennings** [comment]
How about `rick init [path]`? `new` and `create` are possibly too ambiguous about _what_ is new, or _what_ is being created, but `init` is a at least clear that it's the first command that you'll run?
